src/Helper/Translation/TranslationBuilder.php

```diff
@@ -15,23 +15,23 @@ final class TranslationBuilder extends AbstractBuilder
     /**
      * @return \Generator|MessageCatalogue[]
      */
-    public function buildMessageCatalogues(Environment $environment): \Generator
+    public function buildMessageCatalogues(Environment $environment, string $domain = null): \Generator
     {
         if (null === $contentType = $this->getContentType($environment)) {
             return [];
         }
 
         foreach ($this->getMessages($contentType) as $locale => $messages) {
             $messageCatalogue = new MessageCatalogue($locale);
-            $messageCatalogue->add($messages, $environment->getName());
+            $messageCatalogue->add($messages, $domain ?? $environment->getName());
```

Comment:
Should you not use the get alias here?

Reply:
Yes also thought about it, but the [ClientRequest](https://github.com/ems-project/EMSClientHelperBundle/blob/release/3.6/src/Helper/Elasticsearch/ClientRequest.php) getCacheKey uses the environment name. Could not just switch to alias without testing more ..., maybe in a next release